applications/concept-cat/src/main/java/no/ccat/service/ConceptHarvester.java

```diff
@@ -29,22 +27,21 @@
 
     private final ConceptDenormalizedRepository conceptDenormalizedRepository;
     private final RDFToModelTransformer rdfToModelTransformer;
-    private final HarvestAdminClient harvestAdminClient;
     private boolean isRunningForDeveloperLocally = false;
 
-    @PostConstruct
-    @Scheduled(cron = "${application.harvestCron}")
-    public void harvestFromSource() {
-        logger.info("Harvest of Concepts start");
-
-        List<HarvestDataSource> harvestDataSources = this.harvestAdminClient.getDataSources();
-        harvestDataSources.forEach(harvestDataSource ->
-                harvestFromSingleURLSource(harvestDataSource.getUrl(), harvestDataSource.getAcceptHeaderValue())
-        );
-        logger.info("Harvest of Concepts complete");
-    }
-
-    private void harvestFromSingleURLSource(String harvestUri, String acceptHeader) {
+//    @PostConstruct
+//    @Scheduled(cron = "${application.harvestCron}")
+//    public void harvestFromSource() {
+//        logger.info("Harvest of Concepts start");
+//
+//        List<HarvestDataSource> harvestDataSources = this.harvestAdminClient.getDataSources();
+//        harvestDataSources.forEach(harvestDataSource ->
+//                harvestFromSingleURLSource(harvestDataSource.getUrl(), harvestDataSource.getAcceptHeaderValue())
+//        );
+//        logger.info("Harvest of Concepts complete");
+//    }
+
```

Comment:
Don't you still want to call harvester on application start-up?

Reply:
Harvesting is to be done by the application when it start ups. This should not have anything with cron, I would think. After startup, the harvester should sleep until a relevant event is received.